### ISSUE TITLE
Cannot train the same accuracy as the pretrained model

### ISSUE THREAD

**Issue by LiGa0507** [ronghuaiyang/arcface-pytorch#28]
I didn't change the parameters in your code but I can't  train the same accuracy as the pretrained model.Could you tell me the train parameters in "resnet18_110.pth"? Thanks for your help.

Tue May 14 20:19:47 2019 train epoch 49 iter 86 1.9944349123113763 iters/s loss 5.390433311462402 acc 0.359375
Tue May 14 20:20:19 2019 train epoch 49 iter 186 3.1073992095805747 iters/s loss 4.969634532928467 acc 0.375
Tue May 14 20:20:52 2019 train epoch 49 iter 286 3.0889427377963514 iters/s loss 5.293170928955078 acc 0.37890625
Tue May 14 20:21:24 2019 train epoch 49 iter 386 3.084540290424234 iters/s loss 5.502295017242432 acc 0.361328125
Tue May 14 20:21:56 2019 train epoch 49 iter 486 3.1056642862816277 iters/s loss 4.713094234466553 acc 0.373046875
Tue May 14 20:22:29 2019 train epoch 49 iter 586 3.099455363037708 iters/s loss 5.360879421234131 acc 0.349609375
Tue May 14 20:23:01 2019 train epoch 49 iter 686 3.1000786826315307 iters/s loss 5.235039234161377 acc 0.375
Tue May 14 20:23:33 2019 train epoch 49 iter 786 3.081582291481597 iters/s loss 5.106305122375488 acc 0.349609375
(7701, 1024)
total time is 16.355839252471924, average time is 0.06364139786954055
lfw face verification accuracy:  0.8625 threshold:  0.31009173

**Comment by ronghuaiyang** [comment]
the 50 epochs is not enough, try more epochs.
